common-lib/src/cmr/common/xml.clj

```diff
@@ -2,9 +2,8 @@
   "Contains XML helpers for extracting data from XML structs created using clojure.data.xml.
   See the test file for examples."
   (:require [cmr.common.date-time-parser :as p]
-            [cmr.common.services.errors :as errors]
-            [clojure.string :as string]
-            [clojure.java.io :as io])
+            [clojure.string :as string])
```

Comment:
This could be moved over to below the colon.

Reply:
also alphabatized now.